test/TemplateRenderLiquidTest.js

```diff
@@ -704,17 +733,32 @@ test("Liquid Shortcode Multiple Args", async (t) => {
   );
 });
 
-test.skip("Liquid Include Scope Leak", async (t) => {
+test("Liquid Include Scope Leak", async (t) => {
   t.is(
     getNewTemplateRender("liquid", "./test/stubs/").getEngineName(),
     "liquid"
   );
 
-  // This might be by design?
+  // This is by design, `include` assigns value to its parent scope,
+  // use `{% render %}` for separated, clean scope
+  // see: https://github.com/harttle/liquidjs/issues/404#issuecomment-955660149
   let fn = await getNewTemplateRender(
     "liquid",
     "./test/stubs/"
-  ).getCompiledTemplate("<p>{% include scopeleak %}{{ test }}</p>");
+  ).getCompiledTemplate("<p>{% include 'scopeleak' %}{{ test }}</p>");
+  t.is(await fn({ test: 1 }), "<p>22</p>");
+});
+
+test("Liquid Render Scope Leak", async (t) => {
```

Comment:
add a test for `{%render%}` to demonstrate the difference between `render` and legacy `include`.